content/applications/general/email_communication/faq.rst

```diff
@@ -0,0 +1,174 @@
+===
+FAQ 
+===
+
+This document contains an explanation of the most recurring mailing concerns.
+
+We will start by addressing issues of outgoing emails (ex: my client has not received my email), 
+and then, of incoming emails (ex: I do not receive responses from my customers in the database).
+
+Outgoing emails
+===============
+
+.. _red_envelop:
+
+What do you have to check if your email is not sent?
+----------------------------------------------------
+
+The first indicator showing you that the email has not been sent is the red envelope next to the
+date and time of the message.
+
+.. image:: faq/red_envelop.png
+   :align: center
+   :alt: Red envelope displayed in chatter
+
+Common error messages
+~~~~~~~~~~~~~~~~~~~~~
+
+.. _email_communication/daily_limit_mail:
+
+You reached your daily limit: 
+*****************************
+
+.. image:: faq/email_limit.png
+   :align: center
+   :alt: Warning in Odoo upon email limit reached
+
+Each email service provider has its own email sending limits. The limits may be daily, hourly, 
+and sometimes even per minute. This is the same for Odoo, we have to limit our customers to prevent 
+our e-mail servers from being blacklisted.
+
+Here are the default limits for new databases:
+
+- 200 emails/day for Odoo Online and Odoo.sh databases with an active subscription,
+
+- 50 emails/day for one-app free and trial databases,
+
+- in case of migration, your daily limit might be reset to 50 emails a day.
+
+In case you hit the limit, you can:
+
+- Ask our support team to increase your daily limit. We will analyze the situation of your database 
+  depending on (non-exhaustive list):
+
+  - How many users in your database, 
+  - Which apps are installed,
+  - Your bounce rate: the percentage of email addresses that did not receive your emails because 
+      it was returned by a mail server on its way to the final recipient. You can contact 
+      the `support <https://www.odoo.com/help>`_,
```

Comment:
Careful about the continuation lines!
```suggestion
  - Your bounce rate: the percentage of email addresses that did not receive your emails because 
    it was returned by a mail server on its way to the final recipient. You can contact the `support
    <https://www.odoo.com/help>`_.
```